### ISSUE TITLE
expect(block).withArgs is undefined, which I produce the error even following example

### ISSUE THREAD

**Issue by transedward** [mjackson/expect#16]
```js
it('should called withArgs', () => {
    expect(function (check) {
      if (check === 'bad') {
        throw new Error('boom!');
    }
    }).withArgs('bad').toThrow(/boom/);
  })
```
console always thorws 
`TypeError: _expect2.default(...).withArgs is not a function`

and I tried `type of  expect(function (check) {
      if (check === 'bad') {
        throw new Error('boom!');
    }
    })`
I got this: `{ actual: [Function] }`
which means `expect(...).withArgs` is undefined

Something wrong or Am I missing ?

**Comment by rstacruz** [comment]
i think .withArgs is still an unreleased addition... it should be out on the next release.

**Comment by rstacruz** [comment]
@mjackson npm publish?

**Comment by mjackson** [comment]
Thanks for the heads up, @rstacruz. Published in version 1.9.0.
